roles/dtc/common/tasks/ndfc_fabric.yml

```diff
@@ -30,6 +30,14 @@
     src: ndfc_create_fabric.j2
     dest: "{{ role_path }}/files/{{ fabric_file_name }}"
   delegate_to: localhost
+  when: ndfc_version.response.DATA.version is version('12.2.2', '<')
+
+- name: Build Fabric Creation Parameters From Template
+  ansible.builtin.template:
+    src: ndfc_create_fabric_1222.j2
+    dest: "{{ role_path }}/files/{{ fabric_file_name }}"
+  delegate_to: localhost
+  when: ndfc_version.response.DATA.version is version('12.2.2', '>=')
```

Comment:
Couldn't we do this check in the jinja file to limit the tasks (and skips) at runtime?